docs/projects/fps/tutorial.md

```diff
@@ -1,16 +1,18 @@
 # Health pick-up tutorial
 
+![In-game view of the health pickup prefab]({{assetRoot}}assets/health-pickups-tutorial/health-pickup-visible-1.png)
+
 Before starting this tutorial, make sure you have followed the [Get started guide]({{urlRoot}}/get-started).
 
 # What will be covered?
 
-This tutorial implements a simple health pack pickup granting health to players who walk over it. The amount of health granted is fixed and the health pack is consumed through use, respawning a little later.
```

Comment:
use active voice: "As part of this tutorial, we implement ... "

Reply:
Done!